libsession/build.gradle

```
@@ -9,6 +9,9 @@ android {
 
     defaultConfig {
         minSdkVersion androidMinimumSdkVersion
+
+        def useTestnet = project.hasProperty('testnet')
```

Can we have some usage instructions here?

```suggestion
        // To enable testnet usage, from Android Studio select File > Settings > Build, Execution, Deployment > Gradle-Android Compiler
        // and enter a Command-line Option of `-testnet` and then build the application.
        // With testnet enabled the seed node will be http://public.loki.foundation:38157 and any accounts created or messages sent
        // will not be persisted on the network, while on the real network the seed nodes are are follows:
        //  - https://seed1.getsession.org:4443
        //  - https://seed2.getsession.org:4443
        //  - https://seed3.getsession.org:4443
```